CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 0. Configure and install the dependencies: `bundle install`
 0. Make sure the tests pass on your machine: `bundle exec rspec spec`
 0. Create a new branch: `git checkout -b my-branch-name`
-0. Make your change, add tests, and make sure the tests still pass
+0. Make your change, add tests, and make sure the tests still pass and that no warnings are raised
```

Comment:
> This has me stumped

Thanks for trying. I think we can :ship: what we have here if you think it's :+1: 

> I guess another route to go would be to add rubocop.

That would be great! I'm happy to help whittle this down.